### ISSUE TITLE
Region support

### ISSUE THREAD

**Issue by gregorio-disco** [zhw2590582/WFPlayer#55]
Is there any plan to support region selection in the waveform?

**Comment by zhw2590582** [comment]
Do you mean to only display a certain part of the audio waveform?

**Comment by gregorio-disco** [comment]
> Do you mean to only display a certain part of the audio waveform?

No, it's more like selecting a portion (region) of the audio for doing things like annotations. Ref: https://wavesurfer.xyz/examples/?regions.js

**Comment by zhw2590582** [comment]
WFPlayer currently only has simple functions and does not have its own block or region functions, so it does not yet support such complex functions.
